packages/dx-grid-core/src/utils/column.js

```diff
@@ -8,3 +8,13 @@ export const getColumnExtension = (columnExtensions, columnName) => {
   }
   return columnExtension;
 };
+
+export const getColumnExtensionValue = (columnExtensions, extensionName, defaultValue) =>
+  (columnName) => {
+    if (columnExtensions) {
+      const columnExtension = getColumnExtension(columnExtensions, columnName);
```

Comment:
`getColumnExtension` already checks that column extension is defined. Why didn't you add the treed parameter to the `getColumnExtension` method?